src/lib/touistParser.mly

```diff
@@ -230,6 +232,7 @@ expr:
         'A subset B' instead.\n",Some loc);
       Loc (Subset (s1,s2),loc)}
   | p=prop {p}
+  | f=formula_simple {Formula f}
```

Comment:
I just noticed that the new parser has WAY many more conflicts than before (48 shift/reduce and 147 reduce/reduce; before that, it was 7 s/r and 17 r/r):

    Warning: 48 shift/reduce conflicts were arbitrarily resolved.
    Warning: 147 reduce/reduce conflicts were arbitrarily resolved.

All these errors are explained in details in  `_build/src/lib/touistParser.conflicts` (but this file is hard to understand...)

This made me realize that there is something not _sane_ with the way we are doing **formula affectation**: `expr` and `formula_simple` are mixed (thus yielding tons of conflicts)...

Why did we choose to mix `expr` and `formula_simple` in the first place? Can't remember :(